Jenkinsfile

```diff
@@ -55,17 +55,17 @@ try {
           }
         }
 
-        stage('Run instrumented tests') {
-          boolean archiveLog = true
-          String backgroundPid
-          try {
-            backgroundPid = startLogCatCollector()
-            gradle('realm', 'connectedUnitTests')
-            archiveLog = false;
-          } finally {
-            stopLogCatCollector(backgroundPid, archiveLog)
-            storeJunitResults 'realm/realm-library/build/outputs/androidTest-results/connected/**/TEST-*.xml'
-          }
+        stage 'Run instrumented tests'
+        boolean archiveLog = true
+        String backgroundPids = []
```

Comment:
This is not a string anymore. Groovy won't fail and assign the correct type for you, but you might as well use `def`